runtime/README.md

```diff
@@ -58,20 +74,20 @@ Caliptra Runtime Firmware will share driver code with ROM and FMC where
 possible; however, it will have its own copies of all of these drivers linked into
 the Runtime Firmware binary.
 
-## Maibox commands
+## Mailbox commands
 
 All mailbox command codes are little endian.
 
 *Table: Mailbox command result codes*
 
-| **Name**         | **Value**              | Description
-| -------          | -----                  | -----------
-| `SUCCESS`        | `0x0000_0000`          | Mailbox command succeeded
-| `BAD_VENDOR_SIG` | `0x5653_4947` ("VSIG") | Vendor signature check failed
-| `BAD_OWNER_SIG`  | `0x4F53_4947` ("OSIG") | Owner signature check failed
-| `BAD_SIG`        | `0x4253_4947` ("BSIG") | Generic signature check failure (for crypto offload)
-| `BAD_IMAGE`      | `0x4249_4D47` ("BIMG") | Malformed input image
-| `BAD_CHKSUM`     | `0x4243_484B` ("BCHK") | Checksum check failed on input arguments
+| **Name**         | **Value**              | Description                                          |
```

Comment:
nit: I would mildly prefer dropping these table formatting changes from this PR (or splitting them into a separate commit) to make it a bit easier to review the diff from a content perspective. I think they'll render in github the same either way.

But I understand if it's too much trouble to undo.

Reply:
+1 on this

Reply:
Oops! I had reverted this in one of my revisions, but I must have accidentally turned back on table autoformatting. I'll undo these unrelated formatting changes.